tokio-trace/tokio-trace-core/src/field.rs

```diff
@@ -686,8 +686,9 @@ mod test {
     };
 
     impl ::callsite::Callsite for TestCallsite1 {
-        fn add_interest(&self, _: ::subscriber::Interest) {}
-        fn clear_interest(&self) {}
+        fn set_interest(&self, _: ::subscriber::Interest) {
+            unimplemented!()
```

Comment:
I think the reason these were `{}` rather than `unimplemented!()` previously is that calling these functions on the test callsites should not cause the tests to fail.

Do these tests pass with the addition of `unimplemented!()`? Because that's actually kind of surprising to me.

Reply:
they do

Reply:
Okay, that's fine then.